### DISCUSSION TITLE
How to serve one banner to all websites?

### DISCUSSION THREAD

**Discussion by tavoweb** [giuliopons/adadmin#98] [Q&A] [answered]
Is there a way to add one banner and make that served to all added websites ?

**Comment by giuliopons** [comment, marked as answer]
You have to create a Position not binded to a site.
Leave --na-- in the Website field

![image](https://github.com/giuliopons/adadmin/assets/1871627/9ee1c846-56c1-482d-b720-bf572998bf03)

Place the script in the sites where you want the banners to appear.

Advertisers and Admin have to choose that position to send their banner to all sites.

**Reply by tavoweb** [reply]
Then how to solve problem if client like to buy just for one specific website and not add double adwertismennt code ... ?

**Reply by giuliopons** [reply]
At this moment it's not possible to mix the two behaviours:
1) use a position not binded for cross site banners
2) use site specific position for one site banners

**Comment by tavoweb** [comment]
then is not left oportuniti to translate ad to specifik wesite ... ?

**Comment by tavoweb** [comment]
the best be like all positions got thear websites and oportunity to serve banners in all websites ... if u understand what i mean ..

**Reply by giuliopons** [reply]
I think you want use a mix, you want to place a Position on all your sites, and when Advertiser create a banner he decide if he wants to show the banner in just one site, a bunch of sites, or all sites.

This behaviour isn't possible and need a lot of work.

**Comment by tavoweb** [comment]
And i got one more problem im using on some sites in autoload sidebar and
first load its showing advertisement but in next loads then scrolling down
not show banner ...
https://ctl.lt/viskas-ka-jums-reikia-zinoti-apie-miegamojo-lovas/

2023-11-03, pn, 12:52 Giulio Pons ***@***.***> rašė:

> I think you want use a mix, you want to place a Position on all your
> sites, and when Advertiser create a banner he decide if he wants to show
> the banner in just one site, a bunch of sites, or all sites.
>
> This behaviour isn't possible and need a lot of work.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/giuliopons/adadmin/discussions/98#discussioncomment-7465787>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABHB7FZTLHBWMCCWTBFVJ4TYCTEFHAVCNFSM6AAAAAA4UYBEHGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TINRVG44DO>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

*Pagarbiai:*
*Mindaugas Bartusevičius <https://cvman.lt/profile/mindaugas>*
*+37069370376*

**Reply by giuliopons** [reply]
Please start a new post in the Issue section, this problem it's not related to this discussion, it seems related to wrong urls in your code and  problems with http/https
